spec/latest/core/index.html

```diff
@@ -3,6 +3,7 @@
 <head>
 <title>Hydra Core Vocabulary</title>
 <meta charset="utf-8">
+<script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=fetch|gated"></script>
```

Comment:
ReSpec was just updated to accept Promises in the pre- and postprocess hooks. See w3c/respec#1034. Could you please file a separate PR which does that preprocess step (I changed it to postprocess on the weekend before ReSpec was updated). Bonus points for IE support.
We can then discuss the splitting and Markdown conversion separately.
Thanks